lib/indentation-indicator.coffee

```diff
@@ -7,6 +7,9 @@ class IndentationIndicator
     spaceAfterColon:
       type: 'boolean'
       default: false
+    indicatorPosition:
```

Comment:
This should include an `enum` field, ref: https://atom.io/docs/api/latest/Config#enum